src/components/Atoms/Button/Button.css

```diff
@@ -17,9 +17,9 @@
   display: inline-block;
   font: 700 1rem copy;
   letter-spacing: 0.05rem;
-  line-height: 1.5;
+  line-height: 2.25;
```

Comment:
Hmmm, is this going to break these buttons in the header?

Reply:
Probably. Since these override the base buttons, they should be contained to the component js file for this.

Reply:
It doesn't. Header buttons end up being the same height since the padding is being removed from the top and bottom below. Idea is to use line-height to center vertically and add padding so inline usage aligns baseline correctly.